File: Example Code/01 Variables, Data Types, Input and Output/source/Math_FractionMultiplier.java
import java.util.Scanner;

public class Math_Fractions
{    
    public static void main( String args[] )
    {
        Scanner scanner = new Scanner( System.in );
        int num1, denom1, num2, denom2;
        
        System.out.println( "PRODUCT CALCULATOR" );
        System.out.print( "Enter the first fraction numerator & denominator: " );
        num1 = scanner.nextInt();
        denom1 = scanner.nextInt();
        
        System.out.print( "Enter the second fraction numerator & denominator: " );
        num2 = scanner.nextInt();
        denom2 = scanner.nextInt();
        
        int productNum = num1 * num2;
        int productDenom = denom1 * denom2;
        
        System.out.println( "Product: \n" + productNum + "\n---\n" + productDenom );
    }
}

File: Example Code/01 Variables, Data Types, Input and Output/source/Math_FractionMultiplier.cpp
#include <iostream>
using namespace std;

int main()
{
    cout << "FRACTION MULTIPLIER" << endl;
    
    int num1, denom1, num2, denom2;
    
    // Get user input
    cout << "Enter first fraction numerator and denominator: ";
    cin >> num1 >> denom1;
    
    cout << "Enter second fraction numerator and denominator: ";
    cin >> num2 >> denom2;
    
    // Calculate result
    int prodNum, prodDenom;
    prodNum = num1 * num2;
    prodDenom = denom1 * denom2;
    
    // Print result
    cout << endl;  
    cout << num1 << "/" << denom1 << " x "
        << num2 << "/" << denom2 << " = "
        << prodNum << "/" << prodDenom << endl;
    
    return 0;
}
